Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3
+
+WORKDIR /bb8
+COPY bin/requirements.txt /bb8/requirements.txt
+RUN pip3 install -r /bb8/requirements.txt
+
+COPY bin /bb8
+
+COPY secrets /bb8/etc/secrets
+COPY source-config.json /bb8/etc/source-config.json
+
+ARG TARGETS
+RUN python setup.py $TARGETS
+
+ENTRYPOINT ["python", "main.py"]
```

Comment:
As discussed with Rich, we might later split this into a generic bb8 image (up to line 7) built on TeamCity and pushed to docker hub, and a second layer that is built locally as needed to contain configuration-specific stuff.